cardano-db/src/Cardano/Db/Types.hs

```diff
@@ -221,6 +241,7 @@ data PruneConsumeMigration = PruneConsumeMigration
   , -- we make the assumption that if the user is using prune flag
     -- they will also want consume automatically set for them.
     pcmConsumeOrPruneTxOut :: Bool
+  , pcmConsumedTxOut :: Bool
```

Comment:
Remove `pcmConsumedTxOut` and rename `pcmConsumeOrPruneTxOut` to `pcmConsumedTxOut`